ansible/roles/chroma/tasks/main.yaml

```diff
@@ -0,0 +1,112 @@
+---
+- name: "Create user for chroma"
+  user:
+    name: chroma
+    home: "/var/www/chroma"
+    shell: "/usr/sbin/nologin"
+    system: true
+    state: "present"
+
+- name: "Create chroma frontend directory"
+  file:
+    name: "/var/www/chroma/frontend"
+    owner: "chroma"
+    group: "chroma"
+    mode: "2775"
+    state: "directory"
+
+- name: "Create chroma docs directory"
+  file:
+    name: "/var/www/chroma/docs"
+    owner: "chroma"
+    group: "chroma"
+    mode: "2775"
+    state: "directory"
+
+- name: "Create chroma server directory"
+  file:
+    name: "/var/www/chroma/server"
+    owner: "chroma"
+    group: "chroma"
+    mode: "2775"
+    state: "directory"
+
+- name: "Download and extract chroma frontend"
+  unarchive:
+    src: https://github.com/svsticky/chroma/releases/latest/download/frontend.tar.gz
+    dest: "/var/www/chroma/frontend"
+    remote_src: true
+    owner: "chroma"
+    group: "chroma"
+    mode: "775"
+    extra_opts:
+    - "--strip-components=1"
+
+- name: "Download and extract chroma docs"
+  unarchive:
+    src: https://github.com/svsticky/chroma/releases/latest/download/docs.tar.gz
+    dest: "/var/www/chroma/docs"
+    remote_src: true
+    owner: "chroma"
+    group: "chroma"
+    mode: "775"
+    extra_opts:
+      - "--strip-components=1"
+
+- name: "Download and extract chroma server"
+  unarchive:
+    src: https://github.com/svsticky/chroma/releases/latest/download/server-x86_64-unknown-linux-musl
+    dest: "/var/www/chroma/server"
+    remote_src: true
+    owner: "chroma"
+    group: "chroma"
+    mode: "777"
+
+- name: "ensure database user exists"
+  postgresql_user:
+    name: "chroma"
+  become_user: "postgres"
+  become: true
+
+- name: "ensure database exists"
+  postgresql_db:
+    name: "chroma"
+    owner: "chroma"
+  become_user: "postgres"
+  become: true
+
+- name: "ensure database user has the right privileges"
+  postgresql_user:
+    db: "chroma"
+    name: "chroma"
+    priv: "ALL"
+    role_attr_flags: "CREATEDB,LOGIN"
```

Comment:
the choma user should not need to create databases (`CREATEDB`), we already ensure the database exists in the previous step

Reply:
Seems to be the case everywhere.